Alicerce/Sources/Network/Network.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Network.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 06/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public enum Network {
+
+    // MARK: - TypeAlias
+
+    public typealias CompletionClosure = (_ inner: () throws -> Data) -> Void
```

Comment:
i think we do, otherwise it will be `internal` and cannot be used by classes that calls `fetch`